eng/mgmt/automation/utils.py

```diff
@@ -175,6 +178,19 @@ def update_service_ci_and_pom(sdk_root: str, service: str, group: str, module: s
             fout.write(pom_xml)
         logging.info("[POM][Success] Write to pom.xml")
 
+    if os.path.exists(changelog_file):
+        with open(changelog_file, "r") as fin:
+            changelog_str = fin.read()
+    logging.info("[CHANGELOG][Process] Adding initial section to changelog.md..")
+    initial_section = CHANGELOG_INITIAL_SECTION_FORMAT.format(artifact_id=module)
+    if initial_section not in changelog_str:
+        changelog_str += initial_section
+        with open(changelog_file, "w") as fout:
+            fout.write(changelog_str)
```

Comment:
Is this a re-write of the whole file? What about version etc.? Should it be `w+`?